### ISSUE TITLE
If using mergeCells with validateCells function, handsontable is rendering incorrect

### ISSUE THREAD

**Issue by violin0631** [handsontable/handsontable#3903]
### Description
Using mergeCells options and  validateCells() function. See handsontable is rendering incorrect.

### Steps to reproduce
1. Double click in the cell
2. Lost focus input

### Demo
http://jsfiddle.net/09wge8rb/

### Your environment
* Version:  0.28.1
* Browser Name and version: Chrome 54.0.2840.71
* Operating System: Win 10

**Comment by AMBudnik** [comment]
Confirmed.

If we do not define height it is even worse: http://jsfiddle.net/handsoncode/Lfxrny4y/ as extra cells cannot be triggered by scrolling. 

Calling `render()` does not help. 
Adding `rowHeaders` and `colHeaders` does not work either and the first row is gone.

![validate](https://cloud.githubusercontent.com/assets/10757813/20476531/a6aed2fa-afd1-11e6-8ec2-2bb7d6019c87.png)

@violin0631 thanks for sharing.

**Comment by AMBudnik** [comment]
Hi @hjdarnel 

you may try calling `setDataAtCell` (like here http://jsfiddle.net/myqu3rbp/ - fixed after `1000ms`) or any other method that will recalculate the table.

**Comment by kyohei8** [comment]
```
    hot.validateCells();
    setTimeout(() => {
      hot.render();
    }, 0);
```
its another workaround?

**Comment by AMBudnik** [comment]
The issue seems to be solved (used by demo to reproduce) in version 0.37.0.

I am closing this issue but feel free to comment if you still have the same bug in v0.37.0+
